docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,304 @@
+|                        | Description    | $                         |                                 |
+|                        | ************** | ************************  | ************************        |
+|                        | Placement      | IfcLocalPlacement         | IfcCompositeCurve               |
+|                        |                |                           | RepresentationIdentifier="Axis" |
+|                        |                |                           | RepresentationType="Curve2D"    |
+| IfcAlignmentVertical   | Name           | AV1                       |                                 |
+|                        | Description    | $                         |                                 |
+|                        | ************** | ************************  | ************************        |
+|                        | Placement      | IfcLocalPlacement         | IfcGradientCurve                |
+|                        |                |                           | RepresentationIdentifier="Axis" |
+|                        |                |                           | RepresentationType="Curve3D"    |
+
+</details>
+
+<details><summary>Groups Table template</summary>
+
+| **Group** | **Group Type**                | **Minimum** | **Maximum** | **Object**         | **Object Type**      | **Notes** |
+|-----------|-------------------------------|-------------|-------------|--------------------|----------------------|-----------|
+| IfcGroup  | Binari di corsa (Contenitore) | 2           | 2           | IfcFacilityPart    | TRACKSTRUCTURE       | From AL22 |
+| IfcGroup  | Deviatoi                      | 1           | 1           | IfcElementAssembly | TURNOUTPANEL         | From TP01 |
+| IfcGroup  | Massicciata                   | 1           |             | IfcCourse          | BALLASTBED           | From SP01 |
+| IfcGroup  | Traverse                      | 1           |             | IfcGroup           | Segmento di traverse |           |
+| IfcGroup  | Segmento di traverse          | 1           |             | IfcTrackElement    | SLEEPER              | From SP01 |
+| IfcGroup  | Rotaie                        | 1           | 1           | IfcGroup           | Segmento di rotaia   |           |
+| IfcGroup  | Segmento di rotaia            | 2           | 2           | IfcRail            | RAIL                 | From SP01 |
+
+NOTE:
+- when **Minimum** and **Maximum** have the same value, it means exactly. Example: Minimum=Maximum=1, means that the group must groups exactly 1 object of that type.
+- when **Maximum is empty**, it means **unlimited**. Example: Minimum=1; Maximum=empty, means that the group must group 1 or more object of the requested type.
+- the **Notes** column indicates (when the object to be grouped is coming from a pre-requisite test) which is the mentioned test.
+
+</details>
+
```

Comment:
Do we also need an "Assemblies" template?

Reply:
It's coming ;)

Reply:
Object composition (including e.g. Spatial & Element) or separated?

Reply:
Separated as it is with templates to have a clear reference between the documentation and TIs.

Reply:
Noted, I'm adding it to the next commit